src/AngleSharp.Scripting.JavaScript/AngleSharp.Scripting.JavaScript.csproj

```diff
@@ -42,6 +42,9 @@
       <Private>True</Private>
     </Reference>
     <Reference Include="System" />
+    <Reference Include="System.Collections.Concurrent">
+      <HintPath>..\..\..\..\..\..\..\..\..\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETPortable\v4.5\Profile\Profile111\System.Collections.Concurrent.dll</HintPath>
```

Comment:
Yeah I'm not sure why it did like that. I can try removing the hint and see if it builds OK.